prometheus_fzj_weather_exporter/main.py

```diff
@@ -5,9 +5,9 @@
 
 # exporter entry point
 
-# usage:    > python3 main.py --web.listen-address 9184
-# test:     > curl 127.0.0.1:9184
-# (test in a different console or start in background)
+# test usage:
```

Comment:
Nit: the word "test" confuses me here, since it's documenting the normal usage.

I'd be confused about how I'm supposed to run it outside of tests.

Reply:
This change was by one of my first commits, I deleted the whole section because the Usage in the readme is way better. Otherwise I do agree; I just don't think this script needs a copy of it on top.

Reply:
Ahh; I missed that you remove it later.

Sounds good to me. :-)